### PR TITLE
#3: ClientDetector

### LINKED ISSUE #3: ClientDetector
`nmap -sn $NETWORK` where, for example, NETWORK=192.168.0.0/24

### PR DESCRIPTION
**Pull request by KirillSmirnow**

### CODE REVIEW

File: vpn-router-bootstrap/src/main/resources/application.yml [hunk 1]
```diff
@@ -1,2 +1,4 @@
 storage:
   clients-file-path: ${HOME}/vpn-router/storage/clients.yml
+network:
+  address: 192.168.0.0/24
```
Comment (shurshurik) [inline vpn-router-bootstrap/src/main/resources/application.yml:4]: You know my knowledge of computer networks is limited) I googled what 192.168.0.0/24 means. And as I understand, 192.168.0.0 is a standard ip address for many routers and /24 means that we expect ranges between 192.168.0.0 and 192.168.0.255. Am I right?

File: vpn-router-core/src/main/java/vpnrouter/core/infrastructure/shell/ShellExecutor.java [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+
+@Slf4j
+@Component
+public class ShellExecutor {
+
+    @SneakyThrows
+    public List<String> execute(String command) {
+        var process = new ProcessBuilder("sh", "-c", command).start();
```
Comment (shurshurik) [inline vpn-router-core/src/main/java/vpnrouter/core/infrastructure/shell/ShellExecutor.java:17]: wow! I haven't known about such things before)

File: vpn-router-bootstrap/src/main/resources/application.yml (hunk 1 above)
Comment (KirillSmirnow) [reply vpn-router-bootstrap/src/main/resources/application.yml:4]: Yes, you are 👍  That's why I've chosen such network address as the default one.

File: vpn-router-core/src/main/java/vpnrouter/core/infrastructure/shell/ShellExecutor.java (hunk 2 above)
Comment (KirillSmirnow) [reply vpn-router-core/src/main/java/vpnrouter/core/infrastructure/shell/ShellExecutor.java:17]: Hm. You can start subprocesses from Java code :)

File: vpn-router-bootstrap/src/main/resources/application.yml (hunk 1 above)
Comment (shurshurik) [reply vpn-router-bootstrap/src/main/resources/application.yml:4]: 🔥

**Review by shurshurik — APPROVED** [review]

**Pull request merged**